src/Components/Web.JS/src/Services/NavigationManager.ts

```diff
@@ -216,18 +236,12 @@ function ignorePendingNavigation() {
   }
 }
 
-function notifyLocationChanging(uri: string, state: string | undefined, intercepted: boolean): Promise<boolean> {
+function notifyLocationChanging(uri: string, state: string | undefined, intercepted: boolean, callbacks: NavigationCallbacks): Promise<boolean> {
   return new Promise<boolean>(resolve => {
     ignorePendingNavigation();
-
-    if (!notifyLocationChangingCallback) {
-      resolve(false);
-      return;
-    }
```

Comment:
I'm pretty sure this condition was just here so the TypeScript compiler wouldn't complain about `notifyLocationChangingCallback` being undefined. So I don't think we're losing any meaningful behavior by removing this.